packages/office-ui-fabric-react/src/components/List/examples/List.Scrolling.Example.tsx

```diff
@@ -20,6 +20,59 @@ export interface IListScrollingExampleState {
   showItemIndexInView: boolean;
 }
 
+interface IListScrollingExampleClassObject {
```

Comment:
Either remove or rename this interface:
```suggestion
interface IListScrollingExampleClassNames {
```